src/sim-host/ui/sim-host.js

```diff
@@ -18,14 +20,17 @@ customElements.initialize(changePanelVisibilityCallback);
 
 window.addEventListener('DOMContentLoaded', function () {
     sizeContent();
-    Q.all([db.initialize(), initSocketPromise]).then(function () {
-        initializePlugins();
+    Q.all([db.initialize(), initSocketPromise]).then(function (result) {
```

Comment:
I'd suggest using `Q.all([...]).spread(function (initializeIgnored, initResult) {` for the extra documentation value.